### ISSUE TITLE
Cannot position the slider

### ISSUE THREAD

**Issue by marcshilling** [jeanregisser/react-native-slider#2]
I can't seem to do much with this component...

Tried setting all sorts of styles on the style.container object to try to position the slider...`position: 'absolute'`, `width`, `top`, `left`, etc all have no effect...the slider seems to always be the width of the screen and fixed to the top.

**Comment by jeanregisser** [comment]
Hi,

How are you setting the style? Could you post your code?

Right now if you want to position the component you have to pass the `styles` (with a __s__) prop (as seen in https://github.com/jeanregisser/react-native-slider/blob/18ee72ed9983075a85ac8d8ff6d2f9f7bea2e0bc/Example/index.ios.js#L77) and set what you need for the container.
Another possibility is to embed `Slider` in another view like showed in the [README](https://github.com/jeanregisser/react-native-slider/blob/master/README.md).
This displays the slider centered on the screen with a 10px margin on the left and right.

```
<View style={styles.container}>
  <Slider
    value={this.state.value}
    onValueChange={(value) => this.setState({value})} />
  <Text>Value: {this.state.value}</Text>
</View>

var styles = StyleSheet.create({
  container: {
    flex: 1,
    marginLeft: 10,
    marginRight: 10,
    alignItems: 'stretch',
    justifyContent: 'center',
  },
});
```

Anyway I guess I'll change the API so that it directly supports the `style` (without a __s__) prop like you would expect from a standard component.

**Comment by marcshilling** [comment]
Oops! `styles` (plural) is what got me! My bad. Maybe emphasize that in the README?

**Comment by jeanregisser** [comment]
Indeed, I will add this important detail in the README!
I'm still wondering what's the best way to allow users to style different parts.
I guess it's better to have separate props for each distinct part. For instance `trackStyle` and `thumbStyle`. I'll update the code accordingly.
Thanks for reporting.

**Comment by marcshilling** [comment]
Yeah, I like that idea (separate props)!

Things are working great now, thanks for this!

**Comment by jeanregisser** [comment]
FYI, I just published 0.3.0 on npm which supports `style`, `trackStyle` and `thumbStyle` props
(`styles` - plural - still works but will be removed at some point)
